### ISSUE TITLE
[Bug] Can't access config from mobile frontend

### ISSUE THREAD

**Issue by sweebee** [pimatic/pimatic#559]
If i try to open the configuration from the mobile frontend i get this:

```
error [pimatic-mobile-frontend]: Client error: TypeError: undefined is not an object (evaluating '$.mobile.activePage.attr')
```

**Comment by sweetpi** [comment]
Which browser, device?

**Comment by sweebee** [comment]
Looks like it only happened when i have a slow internet connection. Couln't open de config page no matter what i tried. Tried at home and ik worked again. Browser: safari

**Comment by sweebee** [comment]
It only happens when you open the configuration when you haven't yet rebooted after the update was finished. So i think thats the issue.

**Comment by sweetpi** [comment]
Should be fixed in should be fixed in pimatic-mobile-frontend v0.8.52
